tests/cypress/tests/e2e/state-user-dashboard.spec.js

```diff
@@ -0,0 +1,63 @@
+// element selectors
+const enterMcparOnline = '[data-testid="enter-mcpar-online"]';
+const secondButton = '[data-testid="second-enter-mcpar-button"]';
+const addProgram = '[data-testid="add-program"]';
+const enterProgram = '[data-testid="enter-program"]';
+const editProgramButtom = "[data-testid='edit-button']";
+const submitButton = "[data-testid='modal-submit-button']";
+const closeButton = "[data-testid='close-modal']";
+const programNameListItem = "[data-testid='program-name']";
+const leaveFormButton = "[data-testid='leave-form-button']";
```

Comment:
Info: `data-testid` is generally considered the last resort for selecting elements in e2e testing. Wherever possible, it's preferred to use things like the inner text of the button, or a `label` or `role` attribute. These tests could be made more resilient by switching to one of those instead of data-testids. A good rule of thumb is that writing e2e tests should usually not require modifying the tested code.

Read more here:
- [byTestId TestingLibrary docs](https://testing-library.com/docs/queries/bytestid/)
- [Why You Should Avoid Testing React Components With Test IDs](https://betterprogramming.pub/why-you-should-avoid-testing-react-components-with-test-ids-ee50d20d37d2)
- [Making your UI tests resilient to change](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change)

I'd be happy to work through these with you and switch 'em over if you're interested. 👍

Reply:
Definitely interested